SentimentAnalysis/DataPreprocessing.py

```diff
@@ -0,0 +1,16 @@
+import os as os
+import fnmatch as fnmatch
+import re as regex
+
+REPLACE_WITH_NO_SPACE = regex.compile("(\()|(\,)|(\))|(\–)|(\.)|(\;)|(\!)|(\-)|(<br />)|(\:)|(\“)|(\’)|(\‘)|(\”)|(\')")
```

Comment:
Do you only need alphanumerics? If yes, the regex for alphanumeric is better

Reply:
Create a PTR